rosa_getting_started/rosa-quickstart.adoc

```diff
@@ -0,0 +1,156 @@
+include::modules/rosa-getting-started-configure-an-idp.adoc[leveloffset=+2]
+
+.Additional resource
+
+* For detailed steps to configure each of the supported identity provider types, see xref:../rosa_install_access_delete_clusters/rosa-sts-config-identity-providers.adoc#rosa-sts-config-identity-providers[Configuring identity providers for STS]
+
+
+//This content is pulled from rosa-getting-started-grant-user-access.adoc
+include::modules/rosa-getting-started-grant-user-access.adoc[leveloffset=+2]
+
+
+//This content is pulled from rosa-getting-started-grant-admin-privileges.adoc
+include::modules/rosa-getting-started-grant-admin-privileges.adoc[leveloffset=+2]
+
+
+//This content is pulled from rosa-getting-started-access-cluster-web-console.adoc
+include::modules/rosa-getting-started-access-cluster-web-console.adoc[leveloffset=+1]
+
+
+//This content is pulled from deploy-app.adoc
+include::modules/deploy-app.adoc[leveloffset=+1]
+
+
+//This content is pulled from rosa-getting-started-revoking-admin-privileges-and-user-access.adoc
+include::modules/rosa-getting-started-revoking-admin-privileges-and-user-access.adoc[leveloffset=+1]
+
+
+//This content is pulled from rosa-getting-started-revoke-admin-privileges.adoc
+include::modules/rosa-getting-started-revoke-admin-privileges.adoc[leveloffset=+2]
+
+
+//This content is pulled from rosa-getting-started-revoke-admin-privileges.adoc
+include::modules/rosa-getting-started-revoke-user-access.adoc[leveloffset=+2]
+
+
+//This content is pulled from rosa-getting-started-deleting-a-cluster.adoc
+include::modules/rosa-getting-started-deleting-a-cluster.adoc[leveloffset=+1]
+
+[id="next-steps_{context}"]
+== Next steps
+
+* xref:../adding_service_cluster/adding-service.adoc#adding-service[Adding services to a cluster using the {cluster-manager} console]
+* xref:../rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc#rosa-managing-worker-nodes[Managing compute nodes]
+* xref:../rosa_cluster_admin/rosa_monitoring/rosa-configuring-the-monitoring-stack.adoc#rosa-configuring-the-monitoring-stack[Configuring the monitoring stack]
+* xref:../rosa_cluster_admin/rosa_logging/rosa-install-logging.adoc#rosa-install-logging[Installing logging add-on services]
+
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+== Additional resources
+
+* For more information about setting up accounts and ROSA clusters using AWS STS, see xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-sts-overview-of-the-deployment-workflow[Understanding the ROSA with STS deployment workflow]
+
+* For information about setting up accounts and ROSA clusters without using AWS STS, see xref:../rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-getting-started-workflow.adoc#rosa-understanding-the-deployment-workflow[Understanding the ROSA deployment workflow]
+
+* For documentation on upgrading your cluster, see xref:../upgrading/rosa-upgrading.adoc#rosa-upgrading[Upgrading ROSA clusters]
```

Comment:
Super small, but we can put periods at the end of these. On the preview, it's noticeable that they don't have them.